contribute-api.py

```diff
@@ -0,0 +1,32 @@
+import requests
+
+build_id = "8896521"
+chroot = "rhel-9-x86_64"
+url = f"https://logdetective.com/frontend/contribute/copr/{build_id}/{chroot}"
+# url = f"http://localhost:5020/frontend/contribute/copr/{build_id}/{chroot}"
+data = {
+    "username": "FAS:me",
+    "fail_reason": "Failed because...",
+    "how_to_fix": "Like this...",
+    "spec_file": {
+        "name": "llvm.spec",
+        "content": "Yes, actual content of the spec file"
+    },
+    # if you are annotating a container build
+    # "container_file": None,
+    "logs": [
+        {
+            "name": "build.log",
+            "content": "content of the build log",
+            "snippets": [
+                {
+                    "start_index": 1,
+                    "end_index": 2,
+                    "user_comment": "this snippet is relevant because...",
+                    "text": "content of the snippet"
+                }
+            ]
+        }
+    ]
+}
+requests.post(url, json=data)
```

Comment:
Why is this file added here? It looks likes the code from the documentation.

Reply:
That's what I was playing with, I committed it without thinking much about it.